{{cookiecutter.project_slug}}/flake.nix

```diff
@@ -11,7 +11,7 @@
         poetry2nix.overlay
         (final: prev: {
           {{ cookiecutter.project_slug }} = prev.poetry2nix.mkPoetryApplication {
-            python = prev.python39;
+            python = prev.python310;
```

Comment:
Not sure how this comes togheter, but I saw in `test-pr.yaml` version `3.11` being mentioned. Could the version be bumped directly to `3.11`?

Reply:
Good point, maybe we should provide a derivation for both. e.g. as in `pyk`. @goodlyrottenapple, what do you think?